app/src/main/java/com/mapbox/maps/testapp/examples/GesturesActivity.kt

```diff
@@ -37,10 +38,93 @@ class GesturesActivity : AppCompatActivity() {
   private lateinit var mapboxMap: MapboxMap
   private lateinit var gesturesPlugin: GesturesPlugin
   private lateinit var gesturesManager: AndroidGesturesManager
-  private lateinit var gestureAlertsAdapter: GestureAlertsAdapter
+  private val gestureAlertsAdapter: GestureAlertsAdapter = GestureAlertsAdapter()
   private var focalPointLatLng: Point? = null
   private var pointAnnotationManager: PointAnnotationManager? = null
   private lateinit var binding: ActivityGesturesBinding
+  private val rotateListener: OnRotateListener = object : OnRotateListener {
+    override fun onRotateBegin(@NonNull detector: RotateGestureDetector) {
```

Comment:
`@NonNull` is not needed here and below, it's actually visible from `RotateGestureDetector` type and not `RotateGestureDetector?`